src/views/Fridge.vue

```diff
@@ -0,0 +1,98 @@
+                                    <input type="text" class="form-control myInput" placeholder="Ingredient" v-model="ingredientText" name="" id="" aria-describedby="basic-addon1">
+                                    <div class="input-group-prepend">
+                                        <button class="btn btn-success mb-2 add-button" @click.prevent="addIngredient" :disabled="ingredients.length == 3 || !ingredientText">Add</button>
+                                        <button class="btn btn-dark mb-2" v-if="ingredients.length > 0" @click.prevent="deleteAll">Delete All</button>
+                                    </div>
+                                </div>
+                            </form>
+                            <div class="input-group mb-3 ingredients-list" v-for="(ingredient, index) in ingredients" :key="ingredient">
+                                <div class="form-control ingredient-box">
+                                    {{ingredient.words}}
+                                </div>
+                                <div class="input-group-append">
+                                    <span @click.stop="deleteIngredient(index)" class="form-control delete">Delete</span>
+                                </div>
+                            </div>
+                            <button class="btn btn-dark btn-lg btn-block" v-if="ingredients.length > 0">Find my dish!</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Second Half -->
+        <div class="col-md-6 p-0 bg-white h-md-100 loginarea">
+            <div class="d-md-flex align-items-center text-center h-md-100 p-5 justify-content-center">
+                <div class="container">
+                    <div class="justif">
+                    <div class="row">
+                        <p class="firstp">...or just select cousine</p>
+                    </div>
+                    <div class="row">
+                        <p class="seccondp">and we will do the rest!</p>
+                    </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-5 myCol">Product</div>
+                        <div class="col-md-5 myCol">Product</div>
+                        <div class="w-100"></div>
+                        <div class="col-md-5 myCol">Product</div>
+                        <div class="col-md-5 myCol">Product</div>
+                    </div>
+                    <div class="row">
+                        <button class="fourth">Find a random recipe</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+
+    </div>
+    </div>
+</template>
+
+<script>
+export default {
+        name: "Fridge",
+            data() {
+                return{
+                ingredients: [],
+                ingredientText: ''
+                }
+            },
+            methods: {
```

Comment:
Add function like in Home.vue for search recipes, push them to storage and redirect to /results, but use this endpoint instead https://api.spoonacular.com/recipes/findByIngredients